test/functional/feature_dbcrash.py

```diff
@@ -85,7 +85,7 @@ def restart_node(self, node_index, expected_tip):
                 self.nodes[node_index].waitforblock(expected_tip)
                 utxo_hash = self.nodes[node_index].gettxoutsetinfo()['hash_serialized_2']
                 return utxo_hash
-            except:
+            except (JSONRPCException, ConnectionResetError, OSError, ValueError):
```

Comment:
missing `from test_framework.authproxy import JSONRPCException` in `feature_dbcrash.py`

Reply:
Please note that the PR is not ready yet.
Thanks for the review

Reply:
added missing import